docs/flink-getting-started.md

```diff
@@ -683,7 +683,58 @@ env.execute("Test Iceberg DataStream");
+| --------------------------------------------- | --------------------------------------------- | ---------------------------- | ---------------------------------- | ------------------------------------------------------------ |
+| snapshot-id                                   | N/A                                           | N/A                          | N/A                                | For time travel in batch mode. Read data from the specified snapshot-id. |
+| case-sensitive                                | case-sensitive                                | N/A                          | false                              | If true, match column name in a case sensitive way.          |
+| as-of-timestamp                               | N/A                                           | N/A                          | N/A                                | For time travel in batch mode. Read data from the most recent snapshot as of the given time in milliseconds. |
+| connector.iceberg.starting-strategy           | connector.iceberg.starting-strategy           | N/A                          | INCREMENTAL_FROM_LATEST_SNAPSHOT   | Starting strategy for streaming execution. TABLE_SCAN_THEN_INCREMENTAL: Do a regular table scan then switch to the incremental mode. The incremental mode starts from the current snapshot exclusive. INCREMENTAL_FROM_LATEST_SNAPSHOT: Start incremental mode from the latest snapshot inclusive. If it is an empty map, all future append snapshots should be discovered. INCREMENTAL_FROM_EARLIEST_SNAPSHOT: Start incremental mode from the earliest snapshot inclusive. If it is an empty map, all future append snapshots should be discovered. INCREMENTAL_FROM_SNAPSHOT_ID: Start incremental mode from a snapshot with a specific id inclusive. INCREMENTAL_FROM_SNAPSHOT_TIMESTAMP: Start incremental mode from a snapshot with a specific timestamp inclusive. If the timestamp is between two snapshots, it should start from the snapshot after the timestamp. Just for FIP27 Source. |
+| start-snapshot-timestamp                      | N/A                                           | N/A                          | N/A                                | Start to read data from the most recent snapshot as of the given time in milliseconds. |
+| start-snapshot-id                             | N/A                                           | N/A                          | N/A                                | Start to read data from the specified snapshot-id.           |
+| end-snapshot-id                               | N/A                                           | N/A                          | The latest snapshot id             | Specifies the end snapshot.                                  |
+| connector.iceberg.split-size                  | connector.iceberg.split-size                  | read.split.target-size       | Table read.split.target-size       | Target size when combining data input splits.                |
```

Comment:
hint option shouldn't need the prefix for consistency, as there is no naming collision concern.

Default should be the default value from `ScanContext`, not `Table read.split.target-size `

Reply:
Done. Modified to a more specific value. But I'm a little confused, ScanContext is also taken from Table Properties, why can't we just use Table read.split.target-size? There are other places in our documentation that do this, such as spark.
Thx.

Reply:
Users won't see the default value defined in `TableProperties.java`. We already listed table property as a source of the config hierarchy. Saying default as `Table read.split.target-size` can be confusing.

When users read the doc, they want to know what is the actual default value, not `TableProperties.SPLIT_SIZE_DEFAULT`.
```
  public static final long SPLIT_SIZE_DEFAULT = 128 * 1024 * 1024; // 128 MB
```

> If the default value of the table properties changes, we need to synchronize the change here.

I agree with this concern. But I would think those default values are public APIs that we shouldn't really change unless we are making major version change.

Reply:
I see. Thank you very much for your patient explanation.